aws-terraform/README.md

```diff
@@ -1,104 +1,113 @@
 
-* Create a Provision Managers group
-* Create and confirm a Provision Manager user
-* Generate your SCIM bridge credentials
 
-You can then download the `scimsession` file and save your bearer token.  The `scimsession` file contains the credentials for the new Provision Manager user.  This user will create, confirm, and suspend users, and create and manage access to groups.  You should use an email address that is unique.
+## Deployment overview
 
-The bearer token and scimsession file combined can be used to sign in to your Provision Manager account. You’ll need to share the bearer token with your identity provider, but it’s important to **never share it with anyone else**. And never share your scimsession file with **anyone at all**.
+### Required Tools
 
-## OP-SCIM Bridge 
+The minimum supported version of Terraform is `0.12.x`.
 
-The 1Password SCIM Bridge is distributed as a Debian package and installed automatically during the deployment process. The following commands are included in the AWS EC2 instance user-data:  
-```bash
-# configure op-scim repo
-- curl -L https://apt.agilebits.com/gpg.key 2> /dev/null | apt-key add -
-- echo '${SCIM_REPO}' > /etc/apt/sources.list.d/op-scimrepo.list
-# install op-scim
-- apt-get -y -qq update && apt-get -y -qq install op-scim
-# configure op-scim auto update cron job
-- echo '10 * * * * root /usr/local/bin/op-scim-upgrade.sh 2>&1 | logger -t op-scim-deploy-cron' > /etc/cron.d/50_op-scim && chmod 0644 /etc/cron.d/50_op-scim
-```
+You will also want to ensure the `aws` and `terraform` CLI tools are installed for your operating system.
 
-The bridge exposes a service listening on TCP port 3002. The bridge must be deployed and protected by an API gateway, proxy, or load balancer supporting TLS. The bridge service runs using unprivileged user/group credentials, and administrators should use a separate user/group for the bridge service. The bridge writes logs to STDOUT allowing collection with a syslog facility. A systemd unit is included in the package and used to manage service operation, service default environment variables can be altered in `/etc/default/op-scim`. The `scimsession` file and a running instance of Redis are required to start the service. By default, 'localhost' Redis server is configured.  
 
-## Logs 
-Endpoint writes to STDOUT, thus logs can be processed in a preferred way, for example, send to a remote log collector using rsyslog. A systemd unit file configuration can be used to set a specific _SyslogIdentifier_ in order to filter service logs.
+### Instance Size
 
-## DNS
-It is recommended to deploy DNS across the infrastructure to reference cache cluster/nodes and bridge endpoint by FQDN.
+For most installations, one `t3.micro` size instance will be adequate to serve Identity Provider requests. We recommend enabling auto-scaling, but it is not strictly necessary.
```

Comment:
This reference to the size could benefit from a link to the variable in the example

https://github.com/1Password/scim-examples/blob/2c415c731d0b3924c4d247131ec2ae597eafe0f5/aws-terraform/terraform/deploy/example_env/variables.tf#L96